lib/tbot/testhelpers/srv.go

```diff
@@ -159,16 +161,39 @@ func MakeDefaultAuthClient(t *testing.T, log utils.Logger, fc *config.FileConfig
 }
 
 // MakeBot creates a server-side bot and returns joining parameters.
-func MakeBot(t *testing.T, client auth.ClientI, name string, roles ...string) *proto.CreateBotResponse {
+func MakeBot(t *testing.T, client auth.ClientI, name string, roles ...string) (botconfig.OnboardingConfig, *machineidv1pb.Bot) {
```

Comment:
Suggestion: Use a pointer for OnboardingConfig? It's the more "usual" signature.

```suggestion
func MakeBot(t *testing.T, client auth.ClientI, name string, roles ...string) (*botconfig.OnboardingConfig, *machineidv1pb.Bot) {
```

Reply:
[226c20c](https://github.com/gravitational/teleport/pull/37636/commits/226c20cbddd9e6e0a0945f6dd02ecfa32978f643)